### PR TITLE
doc/rados: edit add-or-rm-osds (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit doc/rados/operations/add-or-rm-osds.rst (2 of x).

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/add-or-rm-osds.rst [hunk 1]
```diff
@@ -216,190 +216,200 @@ of the following form:
...
 #. Remove the OSD authentication key:
 
    .. prompt:: bash $
 
       ceph auth del osd.{osd-num}
 
    The value of ``ceph`` for ``ceph-{osd-num}`` in the path is the
-   ``$cluster-$id``.  If your cluster name differs from ``ceph``, use your
-   cluster name instead.
+   ``$cluster-$id``. However, if your cluster name differs from ``ceph``, use
```
Comment (zdover23) [inline doc/rados/operations/add-or-rm-osds.rst:402]: None of this $cluster-$id stuff matches up with the commands that we've provided.

File: doc/rados/operations/add-or-rm-osds.rst [hunk 2]
```diff
@@ -216,190 +216,200 @@ of the following form:
 Starting the OSD
 ----------------
 
-After you add an OSD to Ceph, the OSD is in your configuration. However,
-it is not yet running. The OSD is ``down`` and ``in``. You must start
-your new OSD before it can begin receiving data. You may use
-``service ceph`` from your admin host or start the OSD from its host
-machine:
+After an OSD is added to Ceph, the OSD is in the configuration. However, until
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-osds.rst:219]: "in the configuration"? Maybe "In the cluster"

File: doc/rados/operations/add-or-rm-osds.rst [hunk 3]
```diff
@@ -216,190 +216,200 @@ of the following form:
...
 
-
-Observe the Data Migration
---------------------------
-
-Once you have added your new OSD to the CRUSH map, Ceph  will begin rebalancing
-the server by migrating placement groups to your new OSD. You can observe this
-process with  the `ceph`_ tool. :
+After the new OSD has been added to the CRUSH map, Ceph begins rebalancing the
+server by migrating placement groups (PGs) to the new OSD. To observe this
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-osds.rst:235]: s/server/cluster/

File: doc/rados/operations/add-or-rm-osds.rst [hunk 4]
```diff
@@ -216,190 +216,200 @@ of the following form:
...
+process by using the `ceph`_ tool, run the following command:
 
    .. prompt:: bash $
 
       ceph -w
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-osds.rst:240]: or "watch ceph status"

File: doc/rados/operations/add-or-rm-osds.rst [hunk 5]
```diff
@@ -216,190 +216,200 @@ of the following form:
...
 ======================
 
-When you want to reduce the size of a cluster or replace hardware, you may
-remove an OSD at runtime. With Ceph, an OSD is generally one Ceph ``ceph-osd``
-daemon for one storage drive within a host machine. If your host has multiple
-storage drives, you may need to remove one ``ceph-osd`` daemon for each drive.
-Generally, it's a good idea to check the capacity of your cluster to see if you
-are reaching the upper end of its capacity. Ensure that when you remove an OSD
-that your cluster is not at its ``near full`` ratio.
+It is possible to remove an OSD manually while the cluster is running: you
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-osds.rst:253]: s/while the cluster is running/

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/operations/add-or-rm-osds.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/rados/operations/add-or-rm-osds.rst:402]: These variables show up in `ceph.conf`, where they were fairly obscure.  At this point I think if someone is still using a custom cluster name they aren't running Quincy or Reef anyway

Comment (anthonyeleven) [reply doc/rados/operations/add-or-rm-osds.rst:402]: I would just remove new lines 401-404.

Comment (zdover23) [reply doc/rados/operations/add-or-rm-osds.rst:402]: I agree. Accepted, with thanks.

File: doc/rados/operations/add-or-rm-osds.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-osds.rst:219]: Yeah, that is better. Accepted.

File: doc/rados/operations/add-or-rm-osds.rst (hunk 3 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-osds.rst:235]: Accepted.

File: doc/rados/operations/add-or-rm-osds.rst (hunk 4 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-osds.rst:240]: Added.

File: doc/rados/operations/add-or-rm-osds.rst (hunk 5 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-osds.rst:253]: This is the rare case where I disagree. I think that this "while the cluster is running" clause helps the ignorant reader.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/52450 - Reef backport
https://github.com/ceph/ceph/pull/52451 - Quincy backport

**Pull request merged**